src/inc/bundle.h

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*****************************************************************************
+ **                                                                         **
+ ** bundle.h - Information about applications bundled as a single-file  **
+ **                                                                         **
+ *****************************************************************************/
+
+#ifndef _BUNDLE_H_
+#define _BUNDLE_H_
+
+#include <sstring.h>
+
+class Bundle;
+
+struct BundleLoc
```

Comment:
Nit: I would prefer `BundleFileLocation` or something along those lines. 'loc' to me refers to localization - and there's no need to use abbreviation here. Also "bundle location" is ambiguous - normally I would read that as "the location of the bundle".